### PR TITLE
add information for windows installation to readme

### PR DESCRIPTION
**Pull request by JulianWgs**
- it is necessary to set git checkout Linux style ending before cloning
the repository

I would strongly advise to offer an image hosted on docker hub to avoid these problems in the future. Im already working on an image with alpine, but Im having problems with the apache server.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ Code released under the [MIT License](https://opensource.org/licenses/MIT).
 
 These instructions below should work for Ubuntu 16.04. Probably it will work on other OSes as well with minor modifications.
 
+For Windows: **Before** cloning the repository use `git config --global core.autocrlf false` to enable Linux style line ending. This is necessary for the server to work. Otherwise the error `/usr/bin/env: ‘bash\r’: No such file or directory`  will be shown. After building the image the setting can be set to `true` again.
```
Comment (nmanovic) [inline README.md]: Hi,

Thanks for the PR. We very appreciate that.
Do you think it is a good idea to add comprehensive documentation about the topic? For example, a draft/initial version can be found on OpenCV Q&A forum: http://answers.opencv.org/question/196039/computer-vision-annotation-tool-cvat-for-windows/.

**Comment by JulianWgs** [comment]
I think documentation should live somewhere else. This is just a hint to get minimum viable product working on windows.

I for exampled stumbled across this on my windows laptop. The beauty about docker is that it works cross platform (at least I never had problems before). So I was following the tutorial and it didnt work. My excitement stopped for a second and I didnt try on my ubuntu machine two weeks later, since I thought something in the project was broken. I was actually surprised that it would work on that machine.

The first hurdle on someones own machine should be as small as possible.

**Comment by JulianWgs** [comment]
That would be a much better solution! Please do that.

I kind of wondered why I didnt have this issue before.

**Comment by JulianWgs** [comment]
Are you currently implementing this change in the develepment version?

**Comment by nmanovic** [comment]
@JulianWgs ,

We don't have .gitattributes file in our development version. Need to fix that.

**Comment by nmanovic** [comment]
.gitattributes file was added. The change will be in next version. Please let me know if you still have the problem with latest development version.

**Pull request closed without merging**